### PR TITLE
Disable delegate5.exe and pinvoke3.exe which failed in CI today.

### PR DESCRIPTION
**Pull request by jaykrell**
One was already disabled but that didn't work.

### CODE REVIEW

**Comment by jaykrell** [comment]
Failed in https://github.com/mono/mono/pull/9535.

**Comment by lewurm** [comment]
- instead of putting the failures in Makefile comments, please open a github issue and link it here.
- you should _not_ cherry-pick commits, rather rebase on top of `master`

**Comment by jaykrell** [comment]
It wasn’t in master yet, but otherwise agreed.

**Comment by jaykrell** [comment]
@monojenkins build failed

File: mono/tests/Makefile.am [hunk 1]
```diff
@@ -1538,7 +1538,22 @@ LLVM = $(filter --llvm, $(MONO_ENV_OPTIONS) $(AOT_BUILD_FLAGS))
...
+# Test Result (1 failure / +1)
+# MonoTests.runtime.delegate5.exe
+#
+# https://github.com/mono/mono/issues/9035
+# pinvoke3.exe flaky test
+# https://jenkins.mono-project.com/job/test-mono-pull-request-coop/5360/
+# Test Result (1 failure / +1)
+# MonoTests.runtime.pinvoke3.exe_timedout
+#
+# FIXME sort the lists
```
Comment (lewurm) [inline mono/tests/Makefile.am]: these comments contain duplicated information and it is also hard to update/maintain. This would be enough:
```
delegate5.exe:  https://github.com/mono/mono/issues/9617
pinvoke3.exe: https://github.com/mono/mono/issues/9035
```

FIXME comment is a unrelated change.

**Comment by lewurm** [comment]
delegate5 seems to fail only on arm64 fullaot. pinvoke3 only on coop. please restrict it to those targets.

**Comment by jaykrell** [comment]
Agreed.

**Comment by jaykrell** [comment]
Probably moot or moot enough now. Can revisit as additional failures occur.
